doc/install/easy-install.md

```diff
@@ -47,27 +66,16 @@ conda create -n deepmd deepmd-kit=*=*gpu libdeepmd=*=*gpu lammps cudatoolkit=11.
 ```
 One could change the CUDA Toolkit version from `10.2` or `11.6`.
```

Comment:
Should CUDA Toolkit versions be changed?

Reply:
The CUDA versions in the official channel weren't updated